benchmark/process/next-tick-depth-args.js

```diff
@@ -2,7 +2,7 @@
 
 var common = require('../common.js');
 var bench = common.createBenchmark(main, {
-  millions: [2]
+  millions: [12]
```

Comment:
I always choose something large enough to allow for any re-opts to occur and for results to stabilize. With `2`, I was getting results that varied quite a bit in comparison.